app/routers/agenda.py

```diff
@@ -39,6 +40,18 @@ def agenda(
     """Route for the agenda page, using dates range or exact amount of days."""
 
     user_id = 1  # there is no user session yet, so I use user id- 1.
+
+    start_date_graph, end_date_graph = calc_dates_range_for_agenda(0, 0, 7)
+
+    events_objects_for_graph = agenda_events.get_events_per_dates(
+        db, user_id, start_date_graph, end_date_graph
+    )
+    events_for_graph = {
+        str(start_date_graph + timedelta(i)): 0 for i in range(8)}
```

Comment:
Why 8?

Reply:
well when it was 7 it actually returned 8 days
I changed it to return only a week